src/utils/ShapeInformation.js

```diff
@@ -73,6 +73,11 @@ export type ArgModel = {
   arguments: { [string]: string },
 };
 
+export type ComponentModel = {
+  universe: ShapeUniverse,
+  component: { [string]: string },
```

Comment:
Is there a reason this isn’t this?

```js
component: {props: string},
```

Reply:
Eventually I wanted to maybe add `context` and `state`, so left it open. But I'll change this.